domain/domain.go

```diff
@@ -794,6 +799,40 @@ func (do *Domain) PrivilegeHandle() *privileges.Handle {
 	return do.privHandle
 }
 
+// BindHandler returns the BindInfo.
+func (do *Domain) BindHandler() *bindinfo.Handle {
+	return do.bindHandle
+}
+
+// LoadBindInfoLoop create a goroutine loads BindInfo in a loop, it should be called only once in BootstrapSession.
```

Comment:
// LoadBindInfoLoop create a goroutine loads BindInfo in a loop, it should be
// called only once in BootstrapSession.